src/medCore/process/medAbstractProcess.h

```diff
@@ -32,6 +32,8 @@ class MEDCORE_EXPORT medAbstractProcess : public dtkAbstractProcess
     medAbstractProcess( medAbstractProcess * parent = NULL );
     virtual ~medAbstractProcess();
 
+    using dtkAbstractProcess::setInput;
```

Comment:
I'm not really happy about that one. Our goal is here not to use the methods from dtk that may not be defined in some plugins. Rather, input methods are defined in the processes themselves, allowing more precise setInput parameters (medAbstractImageData or meshes, ...). Making it public may also create ugly warnings of hidden overloaded methods. And now public plugins don't use this setInput methods so you would expect it to do something but in fact it doesn't...

To summarize, I would prefer keeping it private. Don't you have ways to modify your processes and pipelines to keep that one private (it's just probably a story of method renaming) ?